corehq/apps/geospatial/const.py

```diff
@@ -2,6 +2,7 @@
 GPS_POINT_CASE_PROPERTY = 'gps_point'
 
 ALGO_AES = 'aes'
+ALGO_AES_CBC = 'aes-cbc'
```

Comment:
Instead of redefining those const in different apps, can we put it in one place, like `corehq/const.py` ?

Reply:
Agreed with consolidating it. What do you think about keeping it all in `motech/const.py` since that's also the app where the decryption util functions are stored?

Reply:
Sounds good!

Reply:
[1e7f473](https://github.com/dimagi/commcare-hq/pull/35642/commits/1e7f473fd5f1201f7d99b5996f18bc1011168710)